app/connectors/ReferenceDataConnector.scala

```diff
@@ -35,11 +35,11 @@ class ReferenceDataConnector @Inject() (config: FrontendAppConfig, http: HttpCli
     }
   }
 
-  def getCustomsOfficesForCountry(countryCode: CountryCode)(implicit ec: ExecutionContext, hc: HeaderCarrier): Future[Seq[CustomsOffice]] = {
-    val serviceUrl = s"${config.referenceDataUrl}/customs-offices/${countryCode.code}"
+  def getCustomsOfficesOfDepartureForCountry(countryCode: CountryCode)(implicit ec: ExecutionContext, hc: HeaderCarrier): Future[Seq[CustomsOffice]] = {
+    val serviceUrl = s"${config.referenceDataUrl}/customs-offices/${countryCode.code}?role=DEP"
```

Comment:
@ajchilts I see there's a few references to departures (DEP) in this PR. Just want to check if this is correct or a copy and paste error?